Fitness/Controllers/OnboardingGymsViewController.swift

```diff
@@ -0,0 +1,234 @@
+    var nextButtonArrow: UIImageView!
+    var backButton: UIButton!
+    var backButtonArrow: UIImageView!
+    
+    // TableView
+    var gymsTableView: UITableView!
+    let reuseIdentifier = "gymsTableView"
+    
+    // Display Info
+    let tableViewToNext: CGFloat = 94
+    let buttonPadding: CGFloat = 27
+    let buttonHeight: CGFloat = 48
+    let cornerRadius: CGFloat = 5
+    
+    let nextButtonSize: CGFloat = 35
+    let nextButtonPadding = CGSize(width: 40, height: 70)
+    let buttonBorderSize: CGFloat = 2
+    
+    let checkSymbolSize: CGFloat = 24
+    let checkArrowSize = CGSize(width: 16.95, height: 11.59)
+    
+    let maxGymRows = 8
+    let gymCellHeight: CGFloat = 60
+    let gymSidePadding: CGFloat = 16
+    let gymCellVerticalPadding: CGFloat = 14
+    
+    // Gyms
+    var gymNames: [String] = []
+    var favoriteGyms: [String] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .fitnessWhite
+        
+        titleLabel = UILabel()
+        titleLabel.text = "Pick a Gym! Any gym! Cuz we know their Hours!"
+        titleLabel.font = ._24MontserratBold
+        titleLabel.textColor = .fitnessBlack
+        titleLabel.lineBreakMode = .byWordWrapping
+        titleLabel.numberOfLines = 0
+        view.addSubview(titleLabel)
+        
+        gymsTableView = UITableView(frame: .zero, style: .plain)
+        gymsTableView.delegate = self
+        gymsTableView.dataSource = self
+        gymsTableView.register(FavoriteGymCell.self, forCellReuseIdentifier: reuseIdentifier)
+        gymsTableView.isScrollEnabled = false
+        gymsTableView.separatorStyle = .none
+        gymsTableView.clipsToBounds = false
+        view.addSubview(gymsTableView)
+        
+        nextButton = UIButton()
+        nextButton.clipsToBounds = false
+        nextButton.layer.cornerRadius = nextButtonSize / 2
+        nextButton.backgroundColor = .fitnessWhite
+        nextButton.layer.borderColor = UIColor.fitnessMediumGrey.cgColor
+        nextButton.layer.borderWidth = buttonBorderSize
+        nextButton.addTarget(self, action: #selector(goToNextView), for: .touchDown)
+        nextButton.isEnabled = false
+        nextButtonArrow = UIImageView(image: UIImage(named: "arrow"))
+        nextButtonArrow.transform = CGAffineTransform(rotationAngle: CGFloat(Double.pi))
+        nextButton.addSubview(nextButtonArrow)
+        view.addSubview(nextButton)
+        
+        backButton = UIButton()
+        backButton.clipsToBounds = false
+        backButton.layer.cornerRadius = nextButtonSize / 2
+        backButton.backgroundColor = .fitnessWhite
+        backButton.layer.borderColor = UIColor.fitnessMediumGrey.cgColor
+        backButton.layer.borderWidth = buttonBorderSize
+        backButton.addTarget(self, action: #selector(goBackAView), for: .touchDown)
+        backButton.isEnabled = false
+        backButtonArrow = UIImageView(image: UIImage(named: "arrow"))
```

Comment:
See what design thinks about this, but in the habit view controller we used a lighter grey back button without the yellow, not sure if they would want that here too for consistency